packages/editor/src/store/reducer.js

```diff
@@ -273,6 +300,48 @@ const withBlockReset = ( reducer ) => ( state, action ) => {
 	return reducer( state, action );
 };
 
+/**
+ * Higher-order reducer which targets the combined blocks reducer and handles
```

Comment:
What does "handles" entail? It's not immediately obvious to me as a reviewer now, let alone a future hypothetical maintainer.

Reply:
To clarify a bit: This HoR only updates the attributes of blocks not the "byClientId" part but we need to access "byClientId" to know the name of the blocks as we only want to change the `core/block` blocks.

Reply:
I'll add a clarification that this HoR exists because it needs access to both branches of the state tree simultaneously.